### PR TITLE
Added key-value form for stream parameters

### PR DESCRIPTION
**Pull request by lazzy-panda**
**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx [hunk 1]
```diff
@@ -43,13 +36,50 @@ const QueryForm: React.FC<Props> = ({
     formState: { errors },
   } = useForm<FormValues>({
     mode: 'onTouched',
-    resolver: yupResolver(validationSchema),
+    // resolver: yupResolver(validationSchema),
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx:39]: ?

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx [hunk 2]
```diff
@@ -43,13 +36,50 @@ const QueryForm: React.FC<Props> = ({
...
       streamsProperties: '',
     },
   });
 
+  const [properties, setProperties] = useState([{ id: 0, key: '', value: '' }]);
+
+  const onAddProp = () => {
+    const id = properties[properties.length - 1].id + 1;
+    setProperties((p) => [...p, { id, key: '', value: '' }]);
+  };
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx:51]: Can we use `useFieldArray`?

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/__test__/QueryForm.spec.tsx [hunk 3]
```diff
@@ -69,58 +60,10 @@ describe('QueryForm', () => {
...
-      );
-      userEvent.click(screen.getByRole('button', { name: 'Execute' }));
-    });
-    expect(
-      screen.queryByText('streamsProperties is not JSON object')
-    ).not.toBeInTheDocument();
+    setTimeout(() => {
+      expect(screen.getByText('ksql is a required field')).toBeInTheDocument();
+      expect(submitFn).not.toBeCalled();
+    }, 1000);
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/__test__/QueryForm.spec.tsx:66]: Timeout?

File: kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx [hunk 4]
```diff
@@ -43,11 +40,8 @@ describe('Query', () => {
     });
 
     await act(() => {
-      userEvent.paste(
-        within(screen.getByLabelText('KSQL')).getByRole('textbox'),
-        'show tables;'
-      );
-      userEvent.click(screen.getByRole('button', { name: 'Execute' }));
+      fireEvent.paste(screen.getByLabelText('KSQL'), 'show tables;');
+      fireEvent.click(screen.getByRole('button', { name: 'Execute' }));
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx:44]: does userEvent not work here?

File: kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx [hunk 5]
```diff
@@ -65,17 +59,9 @@ describe('Query', () => {
...
-      userEvent.paste(
-        within(
-          screen.getByLabelText('Stream properties (JSON format)')
-        ).getByRole('textbox'),
-        '{"some":"json"}'
-      );
-      userEvent.click(screen.getByRole('button', { name: 'Execute' }));
+      fireEvent.paste(screen.getByLabelText('KSQL'), 'show tables;');
+      fireEvent.paste(screen.getByLabelText('Stream properties:'), 'test');
+      fireEvent.click(screen.getByRole('button', { name: 'Execute' }));
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx:64]: does userEvent not work here?

File: kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx [hunk 6]
```diff
@@ -92,17 +78,9 @@ describe('Query', () => {
...
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx:83]: does userEvent not work here?

File: kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx (hunk 5 above)
Comment (rAzizbekyan) [reply kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx:64]: seems there is a problem related to fieldset tag, via userevent.paste we are getting error in tests but fireevent.paste is working without issues, if there is no mandatory we can let it be fireevent

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/__test__/QueryForm.spec.tsx (hunk 3 above)
Comment (rAzizbekyan) [reply kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/__test__/QueryForm.spec.tsx:66]: fixed using waitFor

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx (hunk 1 above)
Comment (rAzizbekyan) [reply kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx:39]: returned back validation but fields are not required for now, will add it when everything will work properly

**Comment by rAzizbekyan** [comment]
Screen Shot 2022-07-06 at 18 18 36
this is the view we have for now, 
should I change anything regarding how it looks?

**Comment by Haarolean** [comment]
Key/value input fields are too short tbh

**Comment by rAzizbekyan** [comment]
Screen Shot 2022-07-11 at 15 13 43
@Haarolean  how it looks now?

File: kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx (hunk 2 above)
Comment (rAzizbekyan) [reply kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx]: done

**Comment by Haarolean** [comment]
> Screen Shot 2022-07-11 at 15 13 43
> 
> @Haarolean how it looks now?

Better. But I find "key" and "value" placeholders' font size is too small for the inputs and padding from the left side is not enough either.

**Review by workshur — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**